tests/integration/repo_search_test.go

```diff
@@ -58,6 +61,17 @@ func testSearch(t *testing.T, url string, expected []string) {
 	assert.EqualValues(t, expected, filenames)
 }
 
-func executeIndexer(t *testing.T, repo *repo_model.Repository, op func(*repo_model.Repository)) {
-	op(repo)
+func executeIndexer(t *testing.T, repo *repo_model.Repository) {
+	code_indexer.UpdateRepoIndexer(repo)
+
+	for {
```

Comment:
Why we need these code?

Reply:
as the code before just throw the repo in the indexer queue and we never wait for it to be sure it got finished

that way the test is for sure deterministic

an alternative would be to just add a sleep and hope after it the indexer is done (witch normally hapens quite fast (witch is why the test did work in the past))

Reply:
I think the queues on tests are all synchronous.

Reply:
In this case i can rm that ...